main.py

```diff
@@ -1,12 +1,168 @@
+        "13": "Quit"
+        }
+
+    print("Welcome to the RAQIS REELS CLI")
+    print("These are the actions you can perform")
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    return options
+
+def make_choice(options):
+    customer = Customer(url="https://retro-video-store-api.herokuapp.com")
+    video = Video(url="https://retro-video-store-api.herokuapp.com")
+
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        # print("Not a valid selection, please select again")
+        choice = input("Make your selection using the option number: ")
+    
+
+    # if choice in ['7','8','11','12'] and customer.selected_customer == None:
+    #     print("You must select a customer before altering")
+    #     print("Let's select a customer!")
+    #     choice = "10"
+
+    # if choice in ['2','3','5'] and video.selected_video == None:
+    #     print("You must select a video before altering")
+    #     print("Let's select a video!")
+    #     choice = "5"
+
+
+    # if choice in valid_choices:
+    #     print("")
+    #     print("")
+        return choice
+
+def get_video_info():
+    title = input("Please enter title: ")
+    release_date = input("Please enter release_date: ")
+    inventory = input("Please enter total inventory: ")
+    available_inventory = input("Please enter available inventory: ")
+    return title, release_date, inventory, available_inventory
+
+def run_cli(play=True):
+
+    customer = Customer(url="https://retro-video-store-api.herokuapp.com")
+    video = Video(url="https://retro-video-store-api.herokuapp.com")
+
+    options = list_options()
+
+    while play==True:
+        choice = make_choice(options)
+
+        if choice == "1":
+            title = input("Please enter title: ")
+            release_date = input('Enter a date in YYYY-MM-DD format:  ')
+            total_inventory = input("Please enter total inventory: ")
+            print(f"Video profile '{title}' has been created.")   
+            response = video.create_video(title=title, release_date=release_date, total_inventory=int(total_inventory))
+
+        elif choice == "2":
+            video_id = input("Please enter Video ID:  ")
+            title = input("Please enter title: ")
+            release_date = input('Enter a date in YYYY-MM-DD format:  ')
+            total_inventory = input("Please enter total inventory: ")
+            print(f"Video profile '{title}' has been updated.")   
+            response = video.update_video(video_id=video_id, title=title, release_date=release_date, total_inventory=int(total_inventory))
+
+        elif choice == "3":
+            video_id = input("Please enter Video id for deletion: ")
+            response = video.delete_video(video_id=video_id)
+            print(f"Video id '{video_id}' has been deleted")
+
+        elif choice == "4":
+            for video in video.all_videos(): 
```

Comment:
This is actually overwriting the video variable from line 73. Because of that, when I try to edit a video after listing all the videos, I get the following error
```
  File "/Users/jared/Developer/c15/video-store-cli/main.py", line 93, in run_cli
    response = video.update_video(video_id=video_id, title=title, release_date=release_date, total_inventory=int(total_inventory))
AttributeError: 'dict' object has no attribute 'update_video'
```

Reply:
This is a rather odd bug that I wouldn't have expected you to be aware of ahead of time but a helpful takeaway would be to do more manual testing throughout working on the project and again before submitting.